build/controller/valueController.js

```diff
@@ -24,7 +24,7 @@ var __importStar = (this && this.__importStar) || function (mod) {
 };
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.checkCarValue = void 0;
-const calcService = __importStar(require("../services/calcServices"));
+const calcService = __importStar(require("../services/calcServices.js"));
```

Comment:
Is function name _importStar compiled from typescript? can the name be changed to something more descriptive?

Reply:
Yea these are the old build files so its compiled JavaScript/  😂